opcua/server/address_space.py

```diff
@@ -226,8 +232,13 @@ def _add_node(self, item, user):
             desc.IsForward = True
             self._aspace[item.ParentNodeId].references.append(desc)
 
-        # now add our node to db
-        self._aspace[nodedata.nodeid] = nodedata
+            addref = ua.AddReferencesItem()
+            addref.ReferenceTypeId = item.ReferenceTypeId
+            addref.SourceNodeId = nodedata.nodeid
+            addref.TargetNodeId = item.ParentNodeId
+            addref.TargetNodeClass = self._aspace[item.ParentNodeId].attributes[ua.AttributeIds.NodeClass].value.Value.Value
+            addref.IsForward = False
+            self._add_reference(addref, user)
```

Comment:
this is already merged. The diff is outdated. Maybe it get fixed if you rebase over master?